packages/react-components/src/InputRpc/CennznetJsonRpc.ts

```diff
@@ -0,0 +1,39 @@
+import defaultJsonRpc from '@polkadot/jsonrpc';
+import createMethod from '@polkadot/jsonrpc/create/method';
+import createParam from '@polkadot/jsonrpc/create/param';
+import cennznetBare from '@cennznet/api/rpc';
+
+const userRpc = Object.entries(cennznetBare).reduce((user, [sectionName, methods]) => {
+    // @ts-ignore
```

Comment:
Not quite fussy about coding styles, but intent is set to 2 in cennznet-ui, shall we follow on that, and which may also help with fixing ts linting warnings?

Reply:
I was able to address the listing warnings.. however to have same standard will set the intent to 2